chain-signatures/node/src/web/mod.rs

```diff
@@ -121,7 +116,6 @@ async fn state(Extension(state): Extension<Arc<AxumState>>) -> Result<Json<State
     tracing::debug!("fetching state");
     // TODO: rename to last_processed_block when making other breaking changes
     let latest_block_height = state.indexer.last_processed_block().await.unwrap_or(0);
-    let latest_eth_block_height = state.eth_indexer.last_processed_block().await.unwrap_or(0);
     let is_stable = state.indexer.is_stable().await;
```

Comment:
Also now that we're publishing to multiple chains, the `stable` participants is gonna be funky. We need to refactor our code so that the stable check is checked per chain instead of only near chain. Issue created: https://github.com/sig-net/mpc/issues/137